pkg/kubectl/cmd/drain/drain.go

```diff
@@ -84,19 +84,19 @@ const (
 	EvictionKind        = "Eviction"
 	EvictionSubresource = "pods/eviction"
 
-	kDaemonsetFatal      = "DaemonSet-managed pods (use --ignore-daemonsets to ignore)"
-	kDaemonsetWarning    = "Ignoring DaemonSet-managed pods"
-	kLocalStorageFatal   = "pods with local storage (use --delete-local-data to override)"
-	kLocalStorageWarning = "Deleting pods with local storage"
-	kUnmanagedFatal      = "pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet (use --force to override)"
-	kUnmanagedWarning    = "Deleting pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet"
+	daemonsetFatal      = "DaemonSet-managed pods (use --ignore-daemonsets to ignore)"
+	daemonsetWarning    = "Ignoring DaemonSet-managed pods"
+	localStorageFatal   = "pods with local storage (use --delete-local-data to override)"
+	localStorageWarning = "Deleting pods with local storage"
+	unmanagedFatal      = "pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet (use --force to override)"
+	unmanagedWarning    = "Deleting pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet"
```

Comment:
All these messages should start with lower-case, unless that's a resource name, that's a rule we follow in all errors.